File: CS/Form1.cs
using System;
using System.Windows.Forms;
// ...

namespace CustomLink_ListView {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e) {
            ListLink myLink = new ListLink(printingSystem1);

            myLink.ListViewControl = listView1;
            myLink.ShowPreviewDialog();
            
            myLink.Dispose();
        }
    }
}

File: VB/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms
' ...

Namespace CustomLink_ListView
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			Dim myLink As New ListLink(printingSystem1)

			myLink.ListViewControl = listView1
			myLink.ShowPreviewDialog()

			myLink.Dispose()
		End Sub
	End Class
End Namespace